.github/workflows/databases.yml

```diff
@@ -8,51 +8,6 @@ on:
   pull_request:
 
 jobs:
-  mysql_57:
```

Comment:
Why are we removing this? Laravel 11 still supports MySQL 5.7, and I don't think we should be making changes that break on MySQL 5.7.

Reply:
@GrahamCampbell I have already explained on the note above, Laravel 11.x doesn't support MySQL 5.7

https://github.com/laravel/docs/pull/9017

Reply:
I am PRing to revert that. I strongly feel this is too soon.

Reply:
Shouldn't Laravel only support versions of Databases that are supported and secure?

Reply:
@GrahamCampbell Laravel is not the first framework that do this, meanwhile several PRs already merged into framework considering drop support for 5.7 AFAIK.

Reply:
> Shouldn't Laravel only support versions of Databases that are supported and secure?

Just because the OSS vendor is no longer providing support, it doesn't mean that others are not. Many other vendors still backport security fixes for these databases, some even free of charge.

Reply:
Let's continue this discussion on https://github.com/laravel/docs/pull/9113

Reply:
Native support added for renaming columns on MySQL 5.7, we can keep tests.